test/plugins/source/handle-backspace.js

```diff
@@ -0,0 +1,27 @@
+/** @jsx h */
+
+export default (change, editor) => {
+  editor.run('onKeyDown', { key: 'Backspace' })
+  editor.run('onKeyDown', { key: 'Backspace' })
+}
+
+export const input = <value>
+    <document>
+        <p>Before</p>
+        <source>
+            S<cursor/>ome
+            {'\n'}source
```

Comment:
I don't think you should be able to move content between raw xml and normal document.

Reply:
I would like to keep consistency between code block and source elements as much as it is possible so it's easy for users to understand behaviour.